hypothesis/report
# First draft of the required jobs to unify data in report Metabase

For:

 * https://github.com/hypothesis/report/issues/3
 * https://github.com/hypothesis/report/issues/4

## Testing notes

 * Start services in report, h and LMS
 * Run "report/create_from_scratch" SQL task in H then LMS
 * Comment out the `xfail` in `tests/functional/bin/run_sql_task_test.py`
 * This should pass
